File: sample-videochat-conference-java/src/main/java/com/quickblox/sample/videochat/conference/java/utils/WebRtcSessionManager.java
package com.quickblox.sample.videochat.conference.java.utils;

import android.content.Context;

import com.quickblox.conference.ConferenceSession;


public class WebRtcSessionManager {
    private static final String TAG = WebRtcSessionManager.class.getSimpleName();

    private static WebRtcSessionManager instance;
    private Context context;

    private static ConferenceSession currentSession;

    private WebRtcSessionManager(Context context) {
        this.context = context;
    }

    public static WebRtcSessionManager getInstance(Context context) {
        if (instance == null) {
            instance = new WebRtcSessionManager(context);
        }

        return instance;
    }

    public ConferenceSession getCurrentSession() {
        return currentSession;
    }

    public void setCurrentSession(ConferenceSession qbCurrentSession) {
        currentSession = qbCurrentSession;
    }
}

File: sample-videochat-conference-kotlin/src/main/java/com/quickblox/sample/videochat/conference/kotlin/utils/WebRtcSessionManager.kt
package com.quickblox.sample.videochat.conference.kotlin.utils

import com.quickblox.conference.ConferenceSession


object WebRtcSessionManager {

    private var currentSession: ConferenceSession? = null

    fun getCurrentSession(): ConferenceSession? {
        return currentSession
    }

    fun setCurrentSession(qbCurrentSession: ConferenceSession) {
        currentSession = qbCurrentSession
    }
}